### PR TITLE
WiP: (Semi-)automated gnocci installation

### PR DESCRIPTION
**Pull request by nbraud**

### CODE REVIEW

File: ansible/hosts.ini [hunk 1]
```diff
@@ -45,6 +45,8 @@ testvm
 baremetalservers
 virtualservers
 
+[hypervisors]
+alfred
```
Comment (equinox0815) [inline ansible/hosts.ini]: we already have `kvmhosts` group

File: ansible/roles/preseed/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+arch: amd64
```
Comment (equinox0815) [inline ansible/roles/preseed/defaults/main.yml]: variable name prefix please! `arch` is way to generic as a veriable name

File: ansible/roles/vm/install/templates/libvirt-domain.xml.j2 [hunk 3]
```diff
@@ -6,8 +6,8 @@
   <os>
     <type arch='x86_64' machine='pc-0.12'>hvm</type>
 {% if run_installer %}
-    <kernel>{{ vm_host.installer.path }}/{{ vmdistro }}-{{ vmdistcodename }}/{{ hostvars[vmname].vm_install_cooked.arch | default('amd64') }}/linux</kernel>
-    <initrd>{{ vm_host.installer.preseed_path }}/vm-{{ vmname }}-{{ vmdistro }}-{{ vmdistcodename }}.initrd.gz</initrd>
+    <kernel>{{ debian_installer_path }}/{{ distro }}-{{ distcodename }}/{{ hostvars[vmname].vm_install_cooked.arch | default('amd64') }}/linux</kernel>
```
Comment (equinox0815) [inline ansible/roles/vm/install/templates/libvirt-domain.xml.j2]: I would use `install_distro` and `install_distcodename` here as well.

**Comment by nbraud** [comment]
@equinox0815 Yes, this is a work in progress (and I already noticed some of those); I will poke you when it makes sense to look at it.

File: ansible/roles/vm/install/templates/libvirt-domain.xml.j2 (hunk 3 above)
Comment (nbraud) [reply ansible/roles/vm/install/templates/libvirt-domain.xml.j2]: Done

File: ansible/host_vars/gnocci0/main.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+root_password: "{{ vault_root_password }}"
+
+network:
+  nameservers:
+    - 10.12.0.10
+  domain: realraum.at
+  primary:
+    interface: enp1s0
+    ip: 192.168.32.250
```
[the remark this answers is not in the document]
Comment (nbraud) [reply ansible/host_vars/gnocci0/main.yml]: @equinox0815 Could you port it to the new-style network thing?
I can't quite figure out right now what it should look like, for a device that doesn't have a VM host to refer back to.  (It's probably just that I'm tiring out...)

**Pull request merged**